packages/entity-list/src/modules/list/sagas.spec.js

```diff
@@ -305,6 +305,38 @@ describe('entity-list', () => {
           })
         })
 
+        describe('onRowClick saga', () => {
+          it('should not change selection if selectOnRowClick not true', () => {
+            const gen = sagas.onRowClick(actions.onRowClick('1'))
```

Comment:
you could take a look at https://github.com/jfairbank/redux-saga-test-plan
i used it a couple of times and i think it enables better redux-saga testing.